adapters/handlers/rest/configure_api.go

```diff
@@ -752,6 +752,9 @@ func configureAPI(api *operations.WeaviateAPI) http.Handler {
 	}
 	enterrors.GoWrapper(
 		func() {
+			// cleanup unfinished backups on startup
```

Comment:
**suggestion (if-minor):** could we make it configurable via env var, to be able to scale it up and down

Reply:
I don't understand, what would we be configuring?